x-pack/qa/multi-cluster-search-security/src/test/resources/rest-api-spec/test/multi_cluster/80_point_in_time.yml

```
@@ -111,3 +111,39 @@ teardown:
       close_point_in_time:
         body:
           id: "$pit_id"
+
+---
+"Point in time CCS with only remote indices requires no privileges on local cluster":
+
+  - do:
+      headers: { Authorization: "Basic cmVtb3RlOnMza3JpdA==" }
+      open_point_in_time:
+        index: "my_*:point_in_time_index"
+        keep_alive: 5m
+  - set: {id: pit_id}
+
+  - do:
+      headers: { Authorization: "Basic am9lOnMza3JpdA==" }
```

```suggestion
      headers: { Authorization: "Basic cmVtb3RlOnMza3JpdA==" }
```

Not only opening the PIT requires no privs on the local cluster but also using it (and closing), no?